mne_bids/path.py

```diff
@@ -1029,6 +1029,92 @@ def _check_non_sub_ses_entity(bids_path):
     return False
 
 
+def _print_lines_with_na(file, folder, is_tsv, line_numbers, outfile):
+    """Get lines that contain n/a."""
+    na_lines = list()
+    with open(file, 'r', encoding='utf-8-sig') as fid:
+        if is_tsv:  # format tsv files nicely
+            header = _truncate_tsv_line(fid.readline())
+            if line_numbers:
+                header = f'1    {header}'
+            header = header.rstrip()
+        for i, line in enumerate(fid):
+            if 'n/a' in line:
+                if is_tsv:
+                    line = _truncate_tsv_line(line)
+                if line_numbers:
+                    line = str(i + 2) + (5 - len(str(i + 2))) * ' ' + line
+                na_lines.append(line.rstrip())
+    if na_lines:
+        print(op.relpath(file, folder), file=outfile)
+        if is_tsv:
+            print(f'    {header}', file=outfile)
+        if len(na_lines) > 10:
+            na_lines = na_lines[:10]
+            na_lines.append('...')
+        for line in na_lines:
+            print(f'    {line}', file=outfile)
+
+
+def _truncate_tsv_line(line, lim=10):
+    """Truncate a line to the specified number of characters."""
+    return ''.join([str(val) + (lim - len(val)) * ' ' if
+                    len(val) < lim else f'{val[:lim - 1]} '
+                    for val in line.split('\t')])
+
+
+def find_unfilled_fields(folder, line_numbers=True, return_str=False):
+    """Find any unfilled fields in a BIDS directory.
```

Comment:
Could you expand the docstring a little to explain what this does? Just looking at this (and not at the code) I'm totally clueless: what are "fields", and how do they end up in my BIDS directory? What **is** a BIDS directory? Is that my entire dataset? What does "unfilled" mean? Are they empty? Is that bad?

Also I don't think we really use the term "folder" in MNE-BIDS; at least for `BIDSPath.root`, the API docs speak of a "directory"

Reply:
It is searching the BIDS dataset and unfilled fields are any `n/a` entries. `folder` was used in `print_dir_tree` so I copied that

Reply:
I considered changing it from `folder` to `bids_root` but I like the style in `path.py` of keeping functions as general as possible. I updated the docstring to be more informative though but you wouldn't necessarily have to use this on a BIDS dataset.

Reply:
Thanks for the clarifications, @alexrockhill, I really appreciate you taking the time! I'll have another look tomorrow and will try to give you more feedback 👍